addons/mail/static/src/components/discuss/tests/discuss_tests.js

```diff
@@ -1340,8 +1340,8 @@ QUnit.test('basic rendering of message', async function (assert) {
     );
     assert.strictEqual(
         message.querySelector(`:scope .o_Message_authorName`).textContent,
-        "Demo",
-        "should have textually author name in header of message"
+        "Note by Demo",
+        "should have Note by author name in header of message"
```

Comment:
should not change functionally

Reply:
Same as above, just invisible text.

Reply:
Done, changes to tests not required anymore.